framework/Sources/VGSFramework/Core/Enums.swift

```diff
@@ -22,20 +22,20 @@ public enum Environment: String {
 /// - none: Default
 /// - cardNumber: Card number field with formatrion and validation
 /// - dateExpiration: Expiration date with formatting and validation
-/// - cvv: CVV code for card.
+/// - cvc: CVC code for card.
 /// - cardHolderName: Card holder name
 public enum FieldType: Int, CaseIterable {
     case none
     case cardNumber
     case expDate
-    case cvv
+    case cvc
```

Comment:
Why cvc? I know CVV ☹️

Reply:
In US mostly used CVC, same in our JS library, so we make decision to change cvv -> cvc.